src/promptflow-core/promptflow/core/_serving/app_base.py

```diff
@@ -55,7 +56,15 @@ def init_app(self, **kwargs):
         self.sample = get_sample_json(self.project_path, logger)
 
         self.init = kwargs.get("init", {})
-        logger.info("Init params: " + str(self.init))
+        try:
+            init_params = os.environ.get(PROMPTFLOW_FLOW_INIT_CONFIG, "{}")
+            init_dict: dict = json.loads(init_params)
+            init_dict.update(self.init)
```

Comment:
Raise error or warning when user set both `PROMPTFLOW_FLOW_INIT_CONFIG` and `init`?

Reply:
There will be printed logs later, so we won't add them here.

Reply:
what does the logs look like?